optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/resources/META-INF/beans.xml

```diff
@@ -14,12 +14,11 @@
   ~ limitations under the License.
   -->
 
-<beans>
-  <scan>
-    <exclude name="**.client.**"/>
```

Comment:
Hmm, I guess I don't understand what leaking means in this case. @ge0ffrey could you elaborate?